init.lua

```diff
@@ -45,6 +45,12 @@ local function normalize(img, opts)
   return img
 end
 
+-- Set the URL of the listening server
+function M.server(config)
```

Comment:
I am concerned that the name of this function will be confusing. It suggests that the function starts the server.

How about display.use(address)?